cl/tests/providers.py

```diff
@@ -80,3 +80,12 @@ def case_name(self, full: bool = False) -> str:
         plaintiff = self._make_random_party(full)
         defendant = self._make_random_party(full)
         return titlecase(f"{plaintiff} v. {defendant}")
+
+    def random_citation(self) -> str:
```

Comment:
I think the norm here would be `def citation(self) -> str`, right?

Reply:
I think the norm would actually be 

```
  @staticmethod
  def _make_random_citation() -> str:
```

Reply:
sure